recipe/bld.bat

```diff
@@ -21,13 +21,17 @@ set BLD_OPTS=%WIN64% ^
     XERCES_DIR=%LIBRARY_PREFIX% ^
     XERCES_INCLUDE="-I%LIBRARY_INC% -I%LIBRARY_INC%\xercesc" ^
     XERCES_LIB=%LIBRARY_LIB%\xerces-c_3.lib ^
-    HDF4_DIR=%LIBRARY_PREFIX% ^
-    HDF4_LIB="%LIBRARY_LIB%\hdf.lib %LIBRARY_LIB%\mfhdf.lib %LIBRARY_LIB%\xdr.lib" ^
```

Comment:
Thanks for the explanation. I believe that `hdf4` is [*under construction*](https://github.com/conda-forge/staged-recipes/pull/122).

Let's open an issue for that.